src/sql/workbench/contrib/notebook/test/browser/notebookActions.test.ts

```diff
@@ -152,4 +237,207 @@ suite('Notebook Actions', function (): void {
 
 		assert.strictEqual(actualCmdId, NewNotebookAction.INTERNAL_NEW_NOTEBOOK_CMD_ID);
 	});
+
+
+	suite('Kernels dropdown', async () => {
+		let kernelsDropdown: KernelsDropdown;
+		let contextViewProvider: ContextViewProviderStub;
+		let container: HTMLElement;
+		let notebookModel: NotebookModelStub;
+		let configurationService: TestConfigurationService;
+		let notebookEditor: NotebookEditorStub;
+		let sandbox: sinon.SinonSandbox;
+		let setOptionsSpy: sinon.SinonSpy;
+
+		setup(async () => {
+			sandbox = sinon.sandbox.create();
```

Comment:
Is there any particular reason you're using a sandbox here? I haven't seen any need for that for our cases

Reply:
sandbox is an object that makes it easy to restore state back to original (that is to do cleanup). I have used this paradigm in all my previous prs also.

Reply:
Hm, guess I missed that.

Anyways just calling sinon.restore will do the same thing - that just belongs to the root sandbox. There's no need for a separate sandbox the way we're doing it. You'd only use sandboxes if you wanted to group the stubs (for example if you wanted to only restore a subset of them).

Reply:
https://sinonjs.org/releases/v9.0.2/sandbox/

> Since sinon@5.0.0, the sinon object is a default sandbox. Unless you have a very advanced setup or need a special configuration, you probably want to only use that one.

Reply:
ok got it. Removing usage of the sandbox.

Reply:
I tried removing sandbox usage but it seems root sandbox is a feature of sinon in a later version so not available in root repo. I am keeping sandbox usage for now. Separately, I will attempt a sinon upgrade on root repo so hopefully, this won't be needed in the future.